js/src/forum/components/DiscussionPage.js

```diff
@@ -49,6 +50,7 @@ export default class DiscussionPage extends Page {
     app.history.push('discussion');
 
     this.bodyClass = 'App--discussion';
+    this.discussionListClass = DiscussionList;
```

Comment:
What does this do?

Reply:
That was a leftover an intermediate step which we decided to revert again. Good catch, removed in master.